k8s/base/app/deployment.yml

```diff
@@ -19,7 +19,7 @@ spec:
       containers:
         - image: ghcr.io/mlibrary/aspace-containerization/aspace-app:v3.4.1
           name: app
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
I'm changing this here, since we basically have to change it anyway anytime the image gets updated. With that, it could make sense to have a static tag that we use with our deployments, to ensure we don't deploy future development work accidentally to production.